src/Uno.UWP/Media/Playback/MediaPlayer.iOS.cs

```diff
@@ -379,6 +379,12 @@ private void OnSeekCompleted(bool finished)
 		internal void UpdateVideoGravity(AVLayerVideoGravity gravity)
 		{
 			_videoLayer.VideoGravity = gravity;
+			this.Dispose();
```

Comment:
Why do you call Dispose() here?

Reply:
There's no good reason, it must be a leftover from tests. I'll remove it.